team_integration_test.go

```diff
@@ -321,7 +346,7 @@ func TestTeamCreateOptions_Marshal(t *testing.T) {
 	bodyBytes, err := req.BodyBytes()
 	require.NoError(t, err)
 
-	expectedBody := `{"data":{"type":"teams","attributes":{"name":"team name","organization-access":{"manage-policies":true},"visibility":"organization"}}}
+	expectedBody := `{"data":{"type":"teams","attributes":{"name":"team name","organization-access":{"manage-policies":true},"sso-team-id":null,"visibility":"organization"}}}
```

Comment:
I think this change might cause failures when run from CI because the associated feature flag is disabled by default. Should I undo this change?

Reply:
We can bring this back once we remove the feature flag and officially un-beta-fy this field!

Reply:
You can use the `skipIfBeta(t)` helper for tests that require a feature sitting behind a feature flag! I think it still eludes me which feature flags are enabled for CI.